README.md

```diff
@@ -181,6 +181,34 @@ See the [Mailchimp GDPR Tutorial](https://mailchimp.com/help/collect-consent-wit
 #### Retrieving Marketing Permission Keys
 Coming Soon
 
+## Collecting contact events
+
+### Adding an event
+
+To add an event associated with a contact, pass the email, event name and properties (optional) into the `addContactEvent` method. This will add the event to the specified contact.
+Unlike adding or updating contact information. Contact events requests will excecute immediately and will not be reattempted if they fail.
```

Comment:
Should there be a comma instead of period at "information. Contact"?

Reply:
True. Fixed. I also removed the word contact so that it matches the language in the rest of the README